actions/entity/delete.php

```diff
@@ -28,9 +26,12 @@
 $subtype = $entity->getSubtype();
 $container = $entity->getContainerEntity();
 
+$response_data = [
+	'entity' => (array) $entity->toObject(),
```

Comment:
Is the idea here that any action around a thing should return a copy of it? Why the array cast?